test/zeroD/test_zeroD.cpp

```diff
@@ -0,0 +1,67 @@
+#include <cstdio>
+#include <string>
+#include "gtest/gtest.h"
+#include "cantera/thermo.h"
+#include "cantera/kinetics.h"
+#include "cantera/zerodim.h"
+
+using namespace Cantera;
+
+//This test ensures that prior reactor initialization of a reactor does not affect later integration within a network. This example was adapted from test_reactor.py::test_equilibrium_HP.
```

Comment:
@anthony-walker this still looks longer than 88 characters. Also, it doesn't look like you added a space after the comment slashes.

Reply:
@bryanwweber I addressed the inline comments on my local, just wanted to make sure I have the line length correct before I push again. It is showing mine as limited but I will look again to make sure.

Reply:
Make sure you don't have soft wrapping turned on